lib/practice_exercises.rb

```diff
@@ -1,13 +1,49 @@
+require "pry"
+# Time Complexity: o(n)
+# Space Complexity: o(1)
 
-# Time Complexity: ?
-# Space Complexity: ?
+# num = [1, 2, 2, 3, 4, 4]
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  i = 0
+  while i <= list.length
+    if list[i] == list[i + 1]
+      list.delete_at(i)
```

Comment:
Since `delete_at` shifts all the subsequent elements 1 index to the left, it's an O(n) method.  Since you have it in a loop that goes n times this method is O(n<sup>2</sup>)